vtpm-snp/src/vtpm.rs

```diff
@@ -23,14 +23,23 @@ use tss_esapi::tcti_ldr::{DeviceConfig, TctiNameConf};
 use tss_esapi::traits::{Marshall, UnMarshall};
 use tss_esapi::Context;
 
-use crate::hcl;
-
-const SNP_REPORT_SIZE: usize = std::mem::size_of::<AttestationReport>();
 const VTPM_NV_INDEX: u32 = 0x01400001;
```

Comment:
Unrelated to this PR, but we can call this handle, something better than just VTPM_NV_INDEX?